en/ui-system/components/editor/videoplayer.md

```diff
@@ -135,7 +135,13 @@ Please refer to the [VideoPlayer API](../../../api/en/classes/VideoPlayer.html)
 
 ## How to display a UI upon a video
 
-You can check the **stayOnBottom** property on the VideoPlayer in the **Properties** panel
+You can display a UI upon a video in two steps:
+
+1. Make sure the **ENABLE_TRANSPARENT_CANVAS** Macro Config is checked in the **Project Settings**.
```

Comment:
Macro Config
-->
sounds weird when I read the whole sentance outloud.

Reply:
Macro Config this is 

![image](https://user-images.githubusercontent.com/7564028/97952715-553b7200-1dd9-11eb-89d8-7265b4a8dd9c.png)

Reply:
got it. Thank you.

Reply:
1. Make sure the **ENABLE_TRANSPARENT_CANVAS** Macro Config is checked in the **Project Settings**.

Reply:
how does that sound @knoxHuang